spec/features/restaurants/index_page_spec.rb

```diff
@@ -1,18 +1,27 @@
 require 'rails_helper'
 
 feature "Going to Restaurant Index page" do
-    let!(:restaurant1) { FactoryBot.create(:restaurant) }
-    let!(:restaurant2) { FactoryBot.create(:restaurant) }
+  let!(:restaurant1) { FactoryBot.create(:restaurant) }
+  let!(:restaurant2) { FactoryBot.create(:restaurant) }
 
-    scenario "should show a unorder list of restaurants with links and the restaurant names" do
-        visit '/restaurants'
+  scenario "should show a unorder list of restaurants with links and the restaurant names" do
+    visit '/restaurants'
 
-        expect(page).to have_content(restaurant1.name)
-    end
+    expect(page).to have_content(restaurant1.name)
+  end
 
-    scenario "when i click a link to to a restaurant it should take me to their show page" do
-        visit '/restaurants'
-        click_link(restaurant1.name, match: :first)
-        expect(page).to have_current_path("/restaurants/#{restaurant1.id}")
-    end
+  scenario "when i click a link to to a restaurant it should take me to their show page" do
+    visit '/restaurants'
+    click_link(restaurant1.name, match: :first)
+    expect(page).to have_current_path("/restaurants/#{restaurant1.id}")
+  end
+
+  scenario "I am able to delete a restaurant from the index page" do
+    visit '/restaurants'
+
+    expect(page).to have_content('Delete')
+    expect(page).to have_content(restaurant1.name)
+    find(:xpath, "//a[@href='/restaurants/#{restaurant1.id}']", text:'Delete').click
+    expect(page).not_to have_content(restaurant1.name)
+  end
```

Comment:
Can we make this test a bit more robust? Specifically you could:
Check that a success message appears with "restaurant deleted"
Check that the restaurant no longer appears on the index page
(optionally) ensure that the object no longer exists in the database